scripts/build_torch_wheels.sh

```diff
@@ -86,14 +86,17 @@ function maybe_install_sources {
     popd
   fi
 
-  # Check if we have cloned pytorch and cd into the pytorch dir. Within the pytorch
-  # dir there is a subdir `torch`.
-  if [ ! -d "torch" ]; then
+  # Check if we have cloned pytorch or xla and cd into the pytorch dir. 
```

Comment:
Shouldn't the comment be: We are checking "if we ... and cd into the `pytorch` or `pytorch/xla`  dir." 

Also, it may not apply to our cloudbuild triggers, but what happens if we have `torch`, but not `xla` which contains `torch_xla`? The first `pytorch` dir contains `xla` with some starter scripts -- without actual code. 

I don't want to over-convolute the case, so as long as our trigger builds run, I am happy.

Reply:
For the comment, we actually want to end up in the `pytorch` directory after this block. Previously, we clone `pytorch`, `cd` in, then clone `xla`, but stay in `pytorch`. Now, if we are in `xla`, we can `cd ..` to `pytorch`. This way, the change doesn't break any other logic that relies on us being in `pytorch` after this block gets executed.

As for the edge cases, this can definitely still be broken if we run the wheel build script under different circumstances. Hopefully this will be relatively stable. 

The build succeeded, I'm confirming that the duplicate repos aren't present on the new image.